.travis.yml

```diff
@@ -1,43 +1,23 @@
 language: php
 sudo: false
 
-env:
-  global:
-    - deps="no"
-    - SYMFONY_VERSION=""
-
 matrix:
   fast_finish: true
   include:
     - php: 5.3
-    - php: 5.3
-      env: deps="low"
     - php: 5.4
     - php: 5.5
     - php: 5.6
-    - php: 5.6
-      env: SYMFONY_VERSION="2.3.x"
-    - php: 5.6
-      env: SYMFONY_VERSION="2.6.x"
-    - php: 5.6
-      env: SYMFONY_VERSION="2.7.x"
-    - php: 5.6
-      env: SYMFONY_VERSION="2.8.x-dev"
     - php: hhvm
     - php: nightly
   allow_failures:
     - php: hhvm
-    - php: nightly
-    - env: SYMFONY_VERSION="2.7.x-dev"
-    - env: SYMFONY_VERSION="2.8.x-dev"
 
 before_install:
   - composer self-update
```

Comment:
you can even get rid of this (it was necessary to have a Composer version supporting ``--prefer-lowest`` because Travis has not updated its environment since February)